tests/x11/toolkits/prepare.pm

```diff
@@ -43,8 +43,6 @@ sub run {
     zypper_call "in gcc gcc-c++ tcl tk xmessage fltk-devel motif-devel gtk2-devel gtk3-devel java java-devel $qt5_devel";
 
     if (is_opensuse) {
-        zypper_call 'in libqt4-devel' if !is_tumbleweed;
```

Comment:
This will break tests on anything that is not Leap 15.3 and Tumbleweed.

Reply:
I think that toolkits test are not ran for leap15.2,  are they? older things are EOL by now iirc... 

@michaelgrifalconi can you triple check there?

Reply:
@dcermak @foursixnine to my understanding, the only test that runs such code is the `openSUSE Leap 15 ` job group and is based on 15.3 but I am not sure who can confirm this.

Maybe @asdil12 since is the original creator of the test?

Reply:
If qt4 is dropped in leap, there is nothing left where we could check for qt4.
It is not run in sle and tw.

Reply:
thank you!